### PR TITLE
Accommodate KISYS3DMOD prefix

### PR DESCRIPTION
**Pull request by SchrodingersGat**
This PR adjusts rule 10.8 to accept 3D model references that are prefixed with {$KISYS3DMOD} (as generated by KiCad)

### CODE REVIEW

File: pcb/rules/rule10_8.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 from rules.rule import *
 import os
 
+SYSMOD_PREFIX = "{$KISYS3DMOD}/"
```
Comment (jkriege2) [inline pcb/rules/rule10_8.py]: this should be `${KISYS3DMOD}/`
I would say ;-)

**Comment by SchrodingersGat** [comment]
Ugh, whoops. Thanks :)

**Comment by SchrodingersGat** [comment]
@jkriege2 please check this one again when you get a chance )

**Comment by SchrodingersGat** [comment]
Closing as this is dealt with in new KLC

**Pull request closed without merging**